third_party/mkl_dnn/build_defs.bzl

```diff
@@ -14,6 +14,22 @@ def if_mkldnn_openmp(if_true, if_false = []):
         "//conditions:default": if_false,
     })
 
+def if_onednn_v3(if_true, if_false = []):
+    """Returns `if_true` if oneDNN v3.x is used.
+
+    Shorthand for select()'ing on whether we're building with
+    oneDNN v3.x open source library only, without depending on MKL binary form.
```

Comment:
This paragraph seems redundant to the next paragraph. Should we remove it?

Reply:
Yes, removed it.